src/results_uploader/results_uploader.py

```diff
@@ -554,6 +554,11 @@ def main(argv: list[str] | None = None) -> None:
                         and any(path.iterdir())):
                     cts_console_log_dir = path
 
+    # Exit early if there's nothing to upload
+    if not mobly_dirs and not cts_console_log_dir:
+        logging.warning('No logs to upload. Exiting.')
+        exit()
```

Comment:
I'm assuming that having nothing to upload is always an unexpected state. If so, maybe use `logging.error` and a non-zero exit code to signal an error?